File: src/java/com/lezw/nifi/script/samples/ExecuteScript/log/log.py
# Python 记录日志

flowFile = session.get()
if flowFile != None:
    test = flowFile.getAttribute("greeting")
    log.debug(test + ": Debug")
    log.info(test + ": Info")
    log.warn(test + ": Warn")
    log.error(test + ": Error")
    session.transfer(flowFile, REL_SUCCESS)

File: src/java/com/lezw/nifi/script/samples/ExecuteScript/log/log.js
flowFile = session.get();
if (flowFile != null) {
    var test = flowFile.getAttribute("test");
    log.debug(test + ": Debug");
    log.info(test + ": Info");
    log.warn(test + ": Warn");
    log.error(test + ": Error");
    session.transfer(flowFile, REL_SUCCESS);
}
